config/environments/development.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.configure do
+  # Allow requests for all domains e.g. <app>.dev.gov.uk
+  config.hosts.clear
+end
```

Comment:
This should be in the dummy app configuration: https://github.com/alphagov/govuk_publishing_components/blob/master/spec/dummy/config/environments/development.rb 

It's a bit confusing but the code in spec is used every time this app is run in a standalone fashion.

Reply:
Right.. I've just figured it now.